### PR TITLE
Add command options to install dependency.

### PR DESCRIPTION
**Pull request by seyawudba**
--- PR TEMPLATE INSTRUCTIONS (2) ---

This change was necessary to allow users install dependencies seamlessly without any problems.

## Changes
- Extra commands were added to the developer_setup.md file to allow users whose terminals wouldn't execute the dependency install command to also install dependencies without any struggle.

## How I tested this
- A preview was made in the vscode editor to see how the changes would look when pushed.
- The pre-commit checks were also run and everything passed successfully.

## Notes
- I didn't remove the earlier command because I think it might work for users.

## Checklist
- [ ] PR has an informative and human-readable title (this will be pulled into the release notes)
- [ ] Changes are limited to a single goal (no scope creep)
- [ ] Code passed the pre-commit check & code is left cleaner/nicer than when first encountered.
- [ ] Any _change_ in functionality is tested
- [ ] New functions are documented (with a description, list of inputs, and expected output)
- [ ] Placeholder code is flagged / future TODOs are captured in comments
- [ ] Project documentation has been updated if adding/changing functionality.

### CODE REVIEW

File: developer_setup.md [hunk 1]
```diff
@@ -29,6 +29,14 @@ python -m venv ./venv
 pip install .[dev,test]
 ```
 
+Please be informed to use the command below as the square bracket might not be recognized as a pattern.
+
+```shell
+pip install .\[dev,test\]
+
+pip install '.[dev,test]'
```
Comment (elijahbenizzy) [inline developer_setup.md:37]: The second one (single quotes) should always work -- just have that without square brackets?

Comment (seyawudba) [reply developer_setup.md:37]: Could you please come clearer on your question?

Comment (elijahbenizzy) [reply developer_setup.md:35]: There are two pip install commands. Do we need both?

**Review by elijahbenizzy — APPROVED** [review]
Thank you!

File: developer_setup.md (hunk 1 above)
Comment (seyawudba) [reply developer_setup.md:35]: Yes, both of them work but I had to remove one. So I think we are good.

**Pull request merged**